CONTRIBUTING.md

```diff
@@ -25,6 +25,9 @@ search the issues to make sure no one else is working on it.
 1. Don't hesitate to move common functions into uucore if they can be reused by other binaries.
 1. Unsafe code should be documented with Safety comments.
 
+uutils is original code. It cannot contain code from existing GNU or Unix-like
```

Comment:
Could you add this to the list instead of a separate paragraph?

Reply:
Sure!